File: Java/menu.java
import java.util.*;
import java.lang.*;
public class menu{
	public static void main(String args[]){
		items obj = new items();
		Scanner sc = new Scanner(System.in);
		int ch;
		do{
			System.out.println("1. Factorial");
			System.out.println("2. Even or Odd");
			System.out.println("3. Prime or not");
			System.out.println("4. Reverse of a number");
			System.out.println("5. Fibonacci series");
			System.out.println("6. Exit");
			System.out.print("Enter your choice: ");
			ch = sc.nextInt();
			switch(ch){
				case 1: obj.factorial();
						break;
				case 2: obj.EvenorOdd();
						break;
				case 3: obj.isprime();
						break;
				case 4: obj.reverse();
						break;
				case 5: obj.fibonacci();
						break;
				case 6: System.exit(0);
				default: System.out.println("Invalid choice");
			}
		}while(ch!=6);
	}
}
class items{
	void factorial(){
		int n;
		Scanner sc = new Scanner(System.in);
		System.out.print("Enter the number you want me to calculate factorial of: ");
		n = sc.nextInt();
		long fact = 1;
		for(int i = 1; i<=n; i++){
			fact*=i;
		}
		System.out.println("The factorial of "+n+ " is "+fact);
	}
	void EvenorOdd(){
		int n;
		Scanner sc = new Scanner(System.in);
		System.out.print("Enter the number you want to check is even or odd : ");
		n = sc.nextInt();
		if(n%2==0){
			System.out.println(n+" is even");
		}
		else{
			System.out.println(n+" is odd");
		}

	}
	void isprime(){
		Scanner sc = new Scanner(System.in);
		int n;
		System.out.print("Enter the number you want ot check is prime or not: ");
		n = sc.nextInt();
		boolean prime = true;
		for(int i = 2; i<n;i++){
			if(n%i==0){
				prime = false;
				break;
			}
		}
		if(prime){
			System.out.println(n+" is a prime number");
		}
		else{
			System.out.println(n+" is not a prime number");
		}
	}
	void reverse(){
		Scanner sc = new Scanner(System.in);
		int n;
		System.out.print("Enter the number you want to reverse: ");
		n = sc.nextInt();
		int rev = 0;
		while(n!=0){
			rev = rev*10 + n%10;
			n = n/10;
		}
		System.out.println("The reverse of the number is "+rev);
	}
	void fibonacci(){
		Scanner sc = new Scanner(System.in);
		int n;
		System.out.print("Enter the number of terms you want in fibonacci series: ");
		n = sc.nextInt();
		int a = 0, b = 1, c;
		System.out.print(a+" "+b+" ");
		for(int i=2; i<n; i++){
			c = a+b;
			System.out.println(c+" ");
			a=b;
			b=c;
		}
	}

}

File: C++/menu.cpp
#include <iostream>
using namespace std;

class items {
public:
    void factorial() {
        int n;
        cout << "Enter the number you want me to calculate factorial of: ";
        cin >> n;
        long long fact = 1;
        for (int i = 1; i <= n; i++) {
            fact *= i;
        }
        cout << "The factorial of " << n << " is " << fact << endl;
    }

    void EvenorOdd() {
        int n;
        cout << "Enter the number you want to check is even or odd: ";
        cin >> n;
        if (n % 2 == 0) {
            cout << n << " is even" << endl;
        } else {
            cout << n << " is odd" << endl;
        }
    }

    void isprime() {
        int n;
        cout << "Enter the number you want to check is prime or not: ";
        cin >> n;
        bool prime = true;
        for (int i = 2; i < n; i++) {
            if (n % i == 0) {
                prime = false;
                break;
            }
        }
        if (prime) {
            cout << n << " is a prime number" << endl;
        } else {
            cout << n << " is not a prime number" << endl;
        }
    }

    void reverse() {
        int n;
        cout << "Enter the number you want to reverse: ";
        cin >> n;
        int rev = 0;
        while (n != 0) {
            rev = rev * 10 + n % 10;
            n = n / 10;
        }
        cout << "The reverse of the number is " << rev << endl;
    }

    void fibonacci() {
        int n;
        cout << "Enter the number of terms you want in fibonacci series: ";
        cin >> n;
        int a = 0, b = 1, c;
        cout << a << " " << b << " ";
        for (int i = 2; i < n; i++) {
            c = a + b;
            cout << c << " ";
            a = b;
            b = c;
       }
        cout << endl;
    }
	void feesbus() {
		int n;
		cout << "Enter the number to play feesbus if number divisible by 3 then print fees and if divisible by 5 then print bus and if divisible by 3 and 5 then print feesbus: ";
		cin >> n;
		if (n%3==0){
			cout << "fees" << endl;
		}
		else if (n%5==0){
			cout << "bus" << endl;
		}
		else if(n%3==0 && n%5==0){
			cout << "feesbus" << endl;
		}
		else{
			cout << "Check your number " << n << endl;
		}
	}

};

int main() {
    items obj;
    int ch;
    do {
        cout << "1. Factorial" << endl;
        cout << "2. Even or Odd" << endl;
        cout << "3. Prime or not" << endl;
        cout << "4. Reverse of a number" << endl;
        cout << "5. Fibonacci series" << endl;
		    cout << "6. Feesbus" << endl;
        cout << "7. Exit" << endl;
        cout << "Enter your choice: ";
        cin >> ch;
        switch (ch) {
            case 1: obj.factorial(); break;
            case 2: obj.EvenorOdd(); break;
            case 3: obj.isprime(); break;
            case 4: obj.reverse(); break;
            case 5: obj.fibonacci(); break;
            case 6: obj.feesbus(); break;
            case 7: exit(0);
            default: cout << "Invalid choice" << endl;
        }
    } while (ch != 7);
    return 0;
}
